src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/ImportMarketoBulkExtractTablesToPL.sql

```diff
@@ -64,13 +64,14 @@ where
 /* Delta Update MarketoLeads */
 ;with baseMarketoLeadsData as
 (
-		Select distinct MLData.LeadId,MLData.[FirstName],MLData.[LastName],MLData.[EmailAddress],MLData.[CreatedAt],MLData.[UpdatedAt],EmpData.EmployerHashedId,ProviderData.Ukprn As ProviderID
+		Select MLData.LeadId,MLData.[FirstName],MLData.[LastName],MLData.[EmailAddress],MLData.[CreatedAt],MLData.[UpdatedAt],STRING_AGG(EmpData.EmployerHashedId,',') As EmployerHashedId,STRING_AGG(ProviderData.Ukprn,',') As ProviderID
```

Comment:
Can we do the same for update statement please

Reply:
Amend the update stmts